lego/apps/events/models.py

```diff
@@ -783,11 +782,6 @@ class Registration(BasisModel):
     presence = models.CharField(
         max_length=20, default=constants.UNKNOWN, choices=constants.PRESENCE_CHOICES
     )
-    photo_consent = models.CharField(
```

Comment:
Are we sure we want to nuke old consents? Doesn't that conflict with the whole purpose?

Reply:
Yeah, true dat. Keeping them as legacy for now, so that it is easy to remove in like 6 yrs :S. :+1: